lib/distillery/plugins/modify_relup.ex

```diff
@@ -96,12 +96,13 @@ defmodule Releases.Plugin.ModifyRelup do
           {:ok, {mod, chunks}} = :beam_lib.chunks('#{path}', [:attributes])
           {mod, get_in(chunks, [:attributes, :behaviour])}
         end)
-        |> Stream.filter_map(fn {module, behaviours} ->
+        |> Stream.filter(fn({module, behaviours}) ->
```

Comment:
Do not wrap antonymous function arguments with parenthesis please. See https://github.com/lexmag/elixir-style-guide/issues/23 for discussion

Reply:
I did not know that! Thank you for teaching me.